src/helpers/common.ts

```diff
@@ -13,13 +15,14 @@ export const normalizeKeysResult = (result: VerifierLookupResponse) => {
     is_new_key: result.is_new_key,
   };
   if (result && result.keys && result.keys.length > 0) {
-    finalResult.keys = result.keys.map((key) => {
-      return {
-        pub_key_X: key.pub_key_X,
-        pub_key_Y: key.pub_key_Y,
-        address: key.address,
-      };
-    });
+    const finalKey = result.keys[0];
```

Comment:
why are we using only the first key?

Reply:
So there are cases where it is possible that a user can have multiple keys but all the nodes might not have all those shares/pub keys for ex i have seen these cases when a node is temp down but key was assigned successfully by other threshold number of nodes.

and as a result we should only check threshold for first key rather than checking older keys which are not even being used on frontend and can have mismatch in threshold when some nodes have more keys assigned for a user than others.